website/docs/reference/commands/compile.md

```diff
@@ -11,4 +11,22 @@ The `compile` command is useful for:
 2. Manually running compiled SQL. While debugging a model or schema test, it's often useful to execute the underlying `select` statement to find the source of the bug.
 3. Compiling `analysis` files. Read more about analysis files [here](/docs/build/analyses).
 
-It is _not_ a pre-requisite of `dbt run`.
+<VersionBlock firstVersion="1.5">
+
+Starting in dbt v1.5, `compile` becomes interactive in the CLI using the selection syntaxt `--select` or `--inline` to target specic models or Jinja references, respectively. This will return the compiled SQL in the CLI (in addition to the `target/` directory). 
```

Comment:
```suggestion
Starting in dbt v1.5, `compile` becomes interactive in the CLI using the selection syntax `--select` or `--inline` to target specific models or Jinja references, respectively. This will display the compiled SQL in the terminal (in addition to the `target/` directory). 
```